core/profiling/profiling.cpp

```diff
@@ -47,6 +47,31 @@ void godot_init_profiler() {
 	perfetto::Tracing::Initialize(args);
 	perfetto::TrackEvent::Register();
 }
+#elif defined(GODOT_USE_INSTRUMENTS)
+
+#import <os/log.h>
+#import <os/signpost.h>
```

Comment:
Those are already included in the header.

(If you find a way to remove some from the header to reduce the impact of that header on compilation time, that would also be good, but from a quick look I don't see anything obvious - and it's `profiling.h` is only included in `.cpp` files so not too bad.)